projects/core/src/i18n/i18next/i18next-translation.service.ts

```diff
@@ -48,7 +48,7 @@ export class I18nextTranslationService implements TranslationService {
           return;
         }
         if (this.i18next.exists(namespacedKey, options)) {
-          subscriber.next(this.i18next.t(namespacedKey, options));
+          subscriber.next(this.i18next.t(namespacedKey, options) as string);
```

Comment:
[QUESTION]
Why do we have to cast it to string now?

Reply:
had type error, they must have tinkered with the types, still looking at the changelog..

Reply:
I've experimented with the type of the `options` param. When it's `any`, then the return type of the `t()` is some funky type which is not matching `string`.

If only we fix (or workaround) the type of the `options` param, the return type of the `t()` function matches `string`.

So I'd vote for working around the types closer to the root - which would mean in this case casting `options` to type `TOptions` from `i18next`

```diff
- import { i18n } from 'i18next';
+ import { i18n, TOptions } from 'i18next';

- ... this.i18next.t(namespacedKey, options) as string
+ ... this.i18next.t(namespacedKey, options as TOptions)
```

What do you think about it